### PR TITLE
AdapterInterface: promoted all methods from abstract

### PR DESCRIPTION
**Pull request by bryanlatten**

### CODE REVIEW

File: src/AdapterInterface.php [hunk 1]
```diff
@@ -190,18 +201,65 @@ public function fetchPairs( $sql, array $parameters = null, $master = false );
...
   /**
-   * Provided only for backwards compatibility, protects a value being entered
-   * into an SQL statement from command escaping/injection
+   * @param string       $table
+   * @param string       $column
+   * @param string|array $where
+   * @param bool         $master
    *
-   * @param mixed $value
+   * @return mixed|null
```
Comment (be-dmitry) [inline src/AdapterInterface.php]: Is this actually mixed? Doesn't this return numbers as strings anyway?

**Comment by be-dmitry** [comment]
:+1:

**Comment by markdunphy** [comment]
LGTM

File: src/AdapterInterface.php (hunk 1 above)
Comment (bryanlatten) [inline src/AdapterInterface.php]: not anymore - within native mysql preparation, you can get real numbers!

**Pull request merged**